core/chain-configs/src/genesis_config.rs

```diff
@@ -101,6 +101,7 @@ pub struct Genesis {
     pub records: GenesisRecords,
     /// Using zero-size PhantomData is a Rust pattern preventing a structure being constructed
     /// without calling `new` method, which has some initialization routine.
+    #[serde(skip)]
```

Comment:
AFAIU from the serde code this is a no-op, serde writes phantom data as unit struct and serde json throws this away in this specific case (when it is used a field of a struct). Could you verify?

Reply:
I manually verified that with `#[serde(skip)]` this field is actually skipped in the generated file whereas without it, there will be a field `"phantom": null`

Reply:
Thank you